src/main/java/ia/controllers/UpdateCourseMembershipController.java

```diff
@@ -0,0 +1,13 @@
+package ia.controllers;
+
+
+//TEMPORARY PLACEHOLDER CLASS FOR TESTING!!!
```

Comment:
I'm happy to merge with this in it and then it can be updated by whoever is working on the relevant use case.